packages/loading-indicator/src/LoadingIndicator.story.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+
+import { css } from '@leafygreen-ui/emotion';
+import {
+  DarkModeProps,
+  storybookArgTypes,
+  storybookExcludedControlParams,
+  StoryMeta,
+} from '@leafygreen-ui/lib';
+import { spacing } from '@leafygreen-ui/tokens';
+import { Body, InlineCode } from '@leafygreen-ui/typography';
+
+import { DisplayOption } from './Spinner/Spinner.types';
+import { BlobLoader, Spinner } from '.';
+
+export default StoryMeta({
+  title: 'Components/LoadingIndicator',
+  argTypes: {
+    darkMode: storybookArgTypes.darkMode,
+    description: { control: 'text' },
+  },
+  parameters: {
+    default: 'LiveExample',
+    controls: {
+      exclude: storybookExcludedControlParams,
+    },
+  },
+});
+
+const storyRootStyles = css`
+  display: flex;
+  gap: 48px;
+  align-items: end;
+`;
+
+const displayOptionContainerStyles = css`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const labelStyles = css`
+  margin-top: ${spacing[5]}px;
+`;
+
+const Template: ComponentStory<typeof Spinner> = (
+  props: {
+    description?: string;
+  } & DarkModeProps,
```

Comment:
& ResponsiveTypographyProps?

Reply:
Why would it extend ResponsiveTypographyProps?